File: leetcode/reverse-words-in-a-string-iii.java
// https://leetcode.com/problems/reverse-words-in-a-string-iii/

class Solution {
    public String reverseWords(String s) {
        String[] words = s.split(" ");
        StringBuilder word = new StringBuilder();
        
        for(int i = 0; i<words.length; i++){
            StringBuilder sb = new StringBuilder(words[i]);
            word.append(sb.reverse().toString());
            if(i != words.length-1) word.append(" ");
        }
        
        return word.toString();
    }
}

File: leetcode/reverse_words_in_a_string_iii.py
# https://leetcode.com/problems/reverse-words-in-a-string-iii


class Solution:
    def reverseWords(self, s: str) -> str:
        out: str = ""
        for word in s.split():
            for c in range(len(word) - 1, -1, -1):
                out += word[c]
            out += " "
        return out.rstrip()
